ColourCode.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TelecomColorCodeUtility.h"
+
+using namespace std;
+using namespace TelComColorCoder; 
+
+void ValidatePairNumberWithColourPair(int pairNumber,
```

Comment:
The file name should reflect intent of the content. A better name could be chosen for this file, for ex. testColorCode.cpp.

Reply:
updated